### PR TITLE
Working demo for s3 upload feature

### PR DESCRIPTION
**Pull request by maziasy**
NOTE: Please do not merge this - this is a work in progress!
- [x] Created routes for viewing and uploading images using Flask, Boto3 and AWS S3
- [x] Created front-end in React to display, upload and download images from AWS S3 bucket

### CODE REVIEW

File: server/app.py [hunk 1]
```diff
@@ -2,11 +2,47 @@
...
 app.register_blueprint(ping_handler)
+
+s3 = boto3.client(
+	's3',
+	aws_access_key_id=S3_ACCESS_KEY_ID,
+	aws_secret_access_key=S3_SECRET_ACCESS_KEY
+	)
+
+Bootstrap(app)
+app.secret_key = 'secret'
```
Comment (bonnieli) [inline server/app.py]: don't think you need this right

**Review by bonnieli — COMMENTED** [review]
i have no issues with this code!

**Review by john-rigas — APPROVED** [review]
Looks great! We just need to add imports for render_template, boto3, request, flash, and bootstrap, right?

**Comment by john-rigas** [comment]
Looks great to me!

**Pull request closed without merging**